internal/helm/flags/flag.go

```diff
@@ -133,6 +137,10 @@ see https://github.com/kubernetes-sigs/controller-runtime/issues/895 for more in
 		false,
 		"enables secure serving of the metrics endpoint",
 	)
+	flagSet.BoolVar(&f.MetricsAuthNAuthZ,
+		"metrics-authn-authz",
+		false,
+		"enables protection of the metrics endpoint with authn/authz")
```

Comment:
Naming is hard ™️ 

Maybe `metrics-require-rbac`? Somehow, I think we need the flag name to express that this is explicitly RBAC-based authn/authz.

Reply:
Updated to use the suggested.